core/electron/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "iTwin.js ElectronHost and ElectronApp",
   "license": "MIT",
   "engines": {
-    "node": ">=12.22.0 < 14.0 || >=14.17.0 <17.0"
+    "node": ">=12.22.0 < 14.0 || >=14.17.0 <17.0 || >= 18.9.0 < 19.0"
```

Comment:
is there a specific reason to have a hard minimum on 18.9.0? If I'm remembering correctly, we had a hard requirement on a minimum of 14.17 and 12.22 due to the N-API compatibility. I'll look back through the history to see if I can confirm that.

Reply:
No, not for compatibility or API -- It was just the latest version at the time (I think we're up to 18.11) - But I suppose it could be changed to 18.0.

Reply:
done

Reply:
well do we want to support any non LTS versions? I thought the approach we took was we only supported LTS, so thatd be 18.12 (Hydrogen) and up

Reply:
Changed to include versions through 18.